.github/workflows/mraid.yaml

```diff
@@ -0,0 +1,31 @@
+name: Mraid
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
```

Comment:
Do we actually need a trigger on pull request? Only push should be enough for this action

Reply:
not really, I've remove the trigger